cfme/middleware/deployments.py

```diff
@@ -0,0 +1,13 @@
+from cfme.fixtures import pytest_selenium as sel
+from cfme.middleware.provider import HawkularProvider
+
+
+class Deployments(HawkularProvider):
+
+    def __init__(self, name):
+        self.name = name
+        self.provider = self.string_name
+
+    def nav_to_deployments_view(self):
```

Comment:
I haven't seen the UI for this but is the name of the thing Deployments or just Deployment?

Anyway, the nav_to_deployments_view should be rather called load_details to align with the interface of other objects.